src/vm/jitinterface.h

```diff
@@ -1495,8 +1499,10 @@ protected :
 
 #ifdef _TARGET_AMD64_
     BOOL                    m_fAllowRel32;      // Use 32-bit PC relative address modes
-    BOOL                    m_fRel32Overflow;   // Overflow while trying to use encode 32-bit PC relative address. 
-                                                // The code will need to be regenerated with m_fRel32Allowed == FALSE.
+#endif
+#if defined(_TARGET_AMD64_) || defined(_TARGET_ARM64_)
+    BOOL                    m_fJumpStubOverflow;   // Overflow while trying to use encode 32-bit PC relative address.
```

Comment:
Nit: `encode 32-bit PC relative address` is not accurate for ARM64

Reply:
Done